ebs/ebs.go

```diff
@@ -423,7 +434,7 @@ func (d *Driver) DeleteVolume(req Request) error {
 		log.Debugf("Detached %v(%v) from %v", id, volume.EBSID, volume.Device)
 	}
 
-	if !referenceOnly {
+	if !referenceOnly && !volume.PreExist {
```

Comment:
We can differentiate the case at the higher level, or call `convoy delete` directly for some volume we can deem as one-time use. But I don't think it can be judged by created by us or not.